src/Adapter/PlatformServices.WinUI/PlatformServices.WinUI.csproj

```diff
@@ -1,8 +1,4 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <FrameworkIdentifier>NetCore</FrameworkIdentifier>
```

Comment:
Could you give some information (reply here is good) about this change?

Reply:
I just simplified the project, we don't need those anymore. That property is set by the sdk.